client/src/components/friends-page/AddButtonForm.js

```diff
@@ -115,14 +115,17 @@ export default function AddButtonForm({
           placeholder="name goes here..."
           onChange={handleNameChange}
           value={friendName}
+          required
         />
         <input
           className="name-input"
           type="text"
           name="biography"
+          maxLength="125"
```

Comment:
I think this is a good way to limit your users impact on your layout. As long as you inform them about it ;)